### PR TITLE
php{5.x,7.1}-redis 3.1.2

### PR DESCRIPTION
**Pull request by emirb**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by javian** [comment]
Thanks for the submission @emirb ! You need to remove the revision lines alltogether to make this work.

File: Formula/php53-redis.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Php53Redis < AbstractPhp53Extension
   init
   desc "PHP extension for Redis"
   homepage "https://github.com/phpredis/phpredis"
-  url "https://github.com/phpredis/phpredis/archive/3.1.1.tar.gz"
-  sha256 "78c577225daba3282c2968d7b29cb346b9626b2ed3dfddda02e26293e0931995"
+  url "https://github.com/phpredis/phpredis/archive/3.1.2.tar.gz"
+  sha256 "a060fcb7784b9323905cf58557d924a394bb539350bea28d02f910df8ddea1f6"
   revision 2
```
Comment (javian) [inline Formula/php53-redis.rb]: Remove this line

File: Formula/php56-redis.rb [hunk 2]
```diff
@@ -4,8 +4,8 @@ class Php56Redis < AbstractPhp56Extension
   init
   desc "PHP extension for Redis"
   homepage "https://github.com/phpredis/phpredis"
-  url "https://github.com/phpredis/phpredis/archive/3.1.1.tar.gz"
-  sha256 "78c577225daba3282c2968d7b29cb346b9626b2ed3dfddda02e26293e0931995"
+  url "https://github.com/phpredis/phpredis/archive/3.1.2.tar.gz"
+  sha256 "a060fcb7784b9323905cf58557d924a394bb539350bea28d02f910df8ddea1f6"
   revision 2
```
Comment (javian) [inline Formula/php56-redis.rb]: Remove this line

File: Formula/php70-redis.rb [hunk 3]
```diff
@@ -4,8 +4,8 @@ class Php70Redis < AbstractPhp70Extension
   init
   desc "PHP extension for Redis"
   homepage "https://github.com/phpredis/phpredis"
-  url "https://github.com/phpredis/phpredis/archive/3.1.1.tar.gz"
-  sha256 "78c577225daba3282c2968d7b29cb346b9626b2ed3dfddda02e26293e0931995"
+  url "https://github.com/phpredis/phpredis/archive/3.1.2.tar.gz"
+  sha256 "a060fcb7784b9323905cf58557d924a394bb539350bea28d02f910df8ddea1f6"
   revision 2
```
Comment (javian) [inline Formula/php70-redis.rb]: Remove this line

File: Formula/php71-redis.rb [hunk 4]
```diff
@@ -4,8 +4,8 @@ class Php71Redis < AbstractPhp71Extension
   init
   desc "PHP extension for Redis"
   homepage "https://github.com/phpredis/phpredis"
-  url "https://github.com/phpredis/phpredis/archive/3.1.1.tar.gz"
-  sha256 "78c577225daba3282c2968d7b29cb346b9626b2ed3dfddda02e26293e0931995"
+  url "https://github.com/phpredis/phpredis/archive/3.1.2.tar.gz"
+  sha256 "a060fcb7784b9323905cf58557d924a394bb539350bea28d02f910df8ddea1f6"
   revision 2
```
Comment (javian) [inline Formula/php71-redis.rb]: Remove this line

**Comment by emirb** [comment]
@javian Thanks for the feedback, it should be all fine now.

**Comment by javian** [comment]
Thanks @emirb !

**Pull request closed without merging**